### PR TITLE
doc: approtect: minor style edits

### PR DESCRIPTION
**Pull request by greg-fer**
Edited tables to better present the contents.

### CODE REVIEW

File: doc/nrf/app_dev/ap_protect/index.rst [hunk 1]
```diff
@@ -26,16 +26,16 @@ For detailed information, refer to the hardware documentation.
...
 
-   * - Implementation type
+   * - AP-Protect implementation type
      - Default factory state
      - How to enable
      - How to disable
-   * - AP-Protect controlled by hardware
+   * - Hardware
      - Disabled
      - Writing ``UICR.APPROTECT`` to ``Enabled`` and performing reset.
```
Comment (wiba-nordic) [inline doc/nrf/app_dev/ap_protect/index.rst:35]:
```suggestion
     - Writing ``UICR.APPROTECT`` to ``Enabled`` and performing a reset.
```

File: doc/nrf/app_dev/ap_protect/index.rst [hunk 2]
```diff
@@ -26,16 +26,16 @@ For detailed information, refer to the hardware documentation.
...
      - Issuing an ``ERASEALL`` command via CTRL-AP.
        This command erases the flash, UICR, and RAM, including ``UICR.APPROTECT``.
-   * - AP-Protect controlled by hardware and software
+   * - Hardware and software
      - Enabled
      - After it is disabled, a reset or a wake enables the access port protection again.
```
Comment (wiba-nordic) [inline doc/nrf/app_dev/ap_protect/index.rst:40]:
```suggestion
     - When disabled, a reset or a wake enables the access port protection again.
```
This sentence feels like it was taken from a more running text that told how to disable it.

Should consideration be given to the second half of the disabling instructions?

File: doc/nrf/app_dev/ap_protect/index.rst [hunk 3]
```diff
@@ -26,16 +26,16 @@ For detailed information, refer to the hardware documentation.
...
      - Issuing an ``ERASEALL`` command via CTRL-AP.
```
Comment (wiba-nordic) [inline doc/nrf/app_dev/ap_protect/index.rst:41]: Doesn't this erase `UICR.APPROTECT` being programmed to `HwDisabled`, and thus it never stays disabled?

Or am I not understandinf the table correctly?

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

File: doc/nrf/app_dev/ap_protect/index.rst (hunk 3 above)
Comment (greg-fer) [reply doc/nrf/app_dev/ap_protect/index.rst:41]: You erase, then write `UICR.APPROTECT` to `HwDisabled`, program, and avoid using a reset or a wake until you're done working and want to re-enable AP-Protect.

**Comment by greg-fer** [comment]
@wiba-nordic , comments addressed.

**Review by wiba-nordic — APPROVED** [review]

**Review by mbolivar-nordic — APPROVED** [review]

**Pull request merged**